global_vars.tf

```diff
@@ -109,3 +109,19 @@ variable "respondent_account_url" {
 variable "submitted_responses_table_name" {
   description = "Table name of table used for storing Submitted Responses"
 }
+
+# New Relic
+variable "new_relic_enabled" {
+  description = "Enable NewRelic monitoring"
+  default     = false
+}
+
+variable "new_relic_app_name" {
+  description = "NewRelic App Name"
+  default     = "Survey Runner"
+}
+
+variable "new_relic_licence_key" {
+  description = "NewRelic Licence Key"
+  default     = ""
```

Comment:
Any reason this is blank? Don't we have a License key for new relic we can pass?

Reply:
Yes we have a licence key, but i'm not going to push that up to GitHub
If i left it empty it would mean we would need to supply a value even if we didn't have NewRelic enabled